interfaces/apparmor/template_vars.go

```diff
@@ -28,11 +28,11 @@ import (
 
 // templateVariables returns text defining apparmor variables that can be used in the
 // apparmor template and by apparmor snippets.
-func templateVariables(appInfo *snap.AppInfo) []byte {
+func templateVariables(appName string, info *snap.Info) []byte {
 	var buf bytes.Buffer
-	fmt.Fprintf(&buf, "@{APP_NAME}=\"%s\"\n", appInfo.Name)
-	fmt.Fprintf(&buf, "@{SNAP_NAME}=\"%s\"\n", appInfo.Snap.Name())
-	fmt.Fprintf(&buf, "@{SNAP_REVISION}=\"%s\"\n", appInfo.Snap.Revision)
+	fmt.Fprintf(&buf, "@{APP_NAME}=\"%s\"\n", appName)
```

Comment:
It's not used at all as far as I can see (so I'd prefer to remove it all together). @jdstrand or @zyga might have more information, as @zyga recommended I re-use the `APP_NAME` variable here.